backend/community/guides/fields/foldfilefield.py

```diff
@@ -0,0 +1,9 @@
+from drf_base64.fields import Base64FileField
+
+DEFAULT_MAX_FOLD_FILE = 2621440
```

Comment:
Is it used anywhere? + naming is a bit unclear. Is it a file size or what?

Reply:
This whole file is a preparation for input file validation. Will be implemented later.